src/command/local/local.c

```diff
@@ -17,6 +17,18 @@ Local Command
 #include "protocol/helper.h"
 #include "protocol/server.h"
 
+/***********************************************************************************************************************************
+Command handlers
+***********************************************************************************************************************************/
+static const ProtocolServerHandler commandLocalHandlerList[] =
+{
+    {.command = PROTOCOL_COMMAND_ARCHIVE_GET_FILE, .handler = archiveGetFileProtocol},
+    {.command = PROTOCOL_COMMAND_ARCHIVE_PUSH_FILE, .handler = archivePushFileProtocol},
+    {.command = PROTOCOL_COMMAND_BACKUP_FILE, .handler = backupFileProtocol},
+    {.command = PROTOCOL_COMMAND_RESTORE_FILE, .handler = restoreFileProtocol},
+    {.command = PROTOCOL_COMMAND_VERIFY_FILE, .handler = verifyFileProtocol},
+};
```

Comment:
I've added some basic directions in protocol/server.h in 127aaf348a57.

As for local/remote, that really depends on what you are trying to do. The ProtocolServer has no notion of local/remote so that's not the right place to put it. local.h/remote.h could have more info on what is local vs remote, but I don't want to do that as part of this patch. We add new commands so seldom that anything I write will probably be wrong by the time it might be used.